package.json

```diff
@@ -40,10 +40,19 @@
   },
   "devDependencies": {
     "@studion/prettier-config": "^0.1.0",
-    "@types/node": "^22",
-    "prettier": "^3.4.2",
-    "release-it": "^18.1.1",
-    "typescript": "^5.7.3"
+    "@types/async-retry": "^1.4.9",
+    "@types/http-status": "^0.2.30",
+    "@types/node": "^18",
+    "@types/node-fetch": "^2.6.12",
+    "async-retry": "^1.3.3",
+    "http-status": "^2.1.0",
+    "nanospinner": "^1.2.2",
+    "node-fetch": "^2.7.0",
+    "prettier": "^3.0.3",
+    "release-it": "^16.2.1",
```

Comment:
This change is downgrading existing dependencies. Please revert it.

Reply:
Whoops